stdlib/REPL/src/REPL.jl

```diff
@@ -839,7 +839,7 @@ function setup_interface(
         # Transform "foo bar baz" into `foo bar baz` (shell quoting)
         # and pass into Base.repl_cmd for processing (handles `ls` and `cd`
         # special)
-        on_done = respond(repl, julia_prompt) do line
+        on_done = respond(shellmode, repl, julia_prompt, pass_empty=true) do line
```

Comment:
I don't see how this could possibly be correct (and indeed it causes precompilation to fail if I apply it to Julia master).

I would have thought that perhaps something like this might work, but (spoiler alert) it doesn't, it also leads to a precompilation error?!?
```diff
diff --git a/stdlib/REPL/src/REPL.jl b/stdlib/REPL/src/REPL.jl
index 341322ef1a..b1fb44cbe4 100644
--- a/stdlib/REPL/src/REPL.jl
+++ b/stdlib/REPL/src/REPL.jl
@@ -912,6 +912,7 @@ function setup_interface(
         prompt_suffix = hascolor ?
             (repl.envcolors ? Base.input_color : repl.input_color) : "",
         repl = repl,
+        sticky = true,
         complete = ShellCompletionProvider(),
         # Transform "foo bar baz" into `foo bar baz` (shell quoting)
         # and pass into Base.repl_cmd for processing (handles `ls` and `cd`
```